lib/record_store/provider/dynect.rb

```diff
@@ -2,6 +2,10 @@
 
 module RecordStore
   class Provider::DynECT < Provider
+    def self.record_types
+      super.add('ALIAS')
```

Comment:
@es do you think we should keep this as a specialization, or just add ALIAS to the collection of record_types in the base Provider class (now that both providers support it)?